php/php.editor/test/unit/data/testfiles/completion/documentation/issueGH5355.php.testIssueGH5355_01.html

```diff
@@ -0,0 +1,13 @@
+<html><body>
+<pre>Code completion result for source line:
+$this->testMetho|d(null);
+(QueryType=COMPLETION, prefixSearch=false, caseSensitive=true)
+METHOD     testMethod($a)                  [PUBLIC]   ClassName
+</pre><h2>Documentation:</h2><div align="right"><font size=-1></font></div><b>testMethod</b><br/><br/><br />
+<h3>Parameters:</h3>
+<table cellspacing=0 style="border: 0px; width: 100%;">
+<tr><td>&nbsp;</td><td valign="top" style="text-aling:left; border-width: 0px;padding: 1px;padding:3px;" ><nobr></nobr></td><td valign="top" style="text-aling:left; border-width: 0px;padding: 1px;padding:3px;" ><nobr><b>$a</b></nobr></td><td valign="top" style="text-aling:left; border-width: 0px;padding: 1px;padding:3px;width:80%;" >PHPDoc not found</td></tr>
+</table>
+<h3>Returns:</h3>
+<table>
+<tr><th align="left">Type:</th><td>string | int</td></tr></table></body></html>
```

Comment:
Nitpick: Git "warns" here that the file does not end with a new line? Could we add it, please? To all newly added files?

Reply:
Done.